nulog-inference-service/start-nulog-inference.py

```diff
@@ -28,8 +30,18 @@
 ES_ENDPOINT = os.environ["ES_ENDPOINT"]
 ES_USERNAME = os.getenv("ES_USERNAME", "admin")
 ES_PASSWORD = os.getenv("ES_PASSWORD", "admin")
+MINIO_ACCESS_KEY = os.environ["MINIO_ACCESS_KEY"]
```

Comment:
These environment variables will need to change as we are switching over to more general S3 storage and not specifically Minio as introduced in this PR: https://github.com/rancher/opni-inference-service/pull/8

Reply:
I will merge from that PR once it's merged.

Reply:
merged from main, pls review again